pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>OfflineReverseGeocode</groupId>
     <artifactId>OfflineReverseGeocode</artifactId>
     <packaging>jar</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.1-SNAPSHOT</version>
     <name>Offline Reverse Geocoding Java library</name>
     <url>http://github.com/AReallyGoodName/OfflineReverseGeocode</url>
```

Comment:
I'm not sure what the protocol for this is, but we might want to update this to reference our own repository. 

Might also be some scm configurations needed to work with our maven release and jenkins structure. 

And also we might want to bring in the fts parent pom for things like checkstyle

Reply:
Yeah, I wasn't sure either. I know we did something similar for hammock, so I'll see how we handled it there.

Reply:
I added the global_pom. I checked the other forked repositories, but we didn't change the `<url>` value for any of those. I do still need to get the SCM stuff setup for building.